serving/docs/configurations_large_model_inference_containers.md

```diff
@@ -0,0 +1,54 @@
+| option.enable_cuda_graph  | No       | Activates capturing the CUDA graph of the forward pass to accelerate.                                                                                                                           | `True`                         |
+| option.triangular_masking | No       | Whether to use triangular masking for the attention mask. This is application or model specific.                                                                                                | `False`                        |
+| option.return_tuple       | No       | Whether transformer layers need to return a tuple or a tensor.                                                                                                                                  | `False`                        |
+| option.training_mp_size   | No       | If the model was trained with DeepSpeed, this indicates the tensor parallelism degree with which the model was trained. Can be different than the tensor parallel degree desired for inference. | `2`                            |
+| option.checkpoint         | No       | Path to DeepSpeed compatible checkpoint file.                                                                                                                                                   | `ds_inference_checkpoint.json` |
+
+### FasterTransformer ([doc](https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-configuration.html))
+
+[Common settings](#common-doc)
+
+### HuggingFace ([doc](https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-configuration.html))
+
+| Item                                    | Required | Description                                                                                                                                                                        | Example value                                            |
+|-----------------------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| option.task                             | No       | The task used in Hugging Face for different pipelines.                                                                                                                             | `text-generation`                                        |
+| option.device_id                        | No       | Load model on a specific device. Do not set this if you use option.tensor_parallel_device.                                                                                         | `0, 1`                                                   |
```

Comment:
We don't have this option, are you referring to `load_on_devices`?
device_id is used internally and should not be set manually.

Reply:
Replaced. This is documented in `Hugging Face handler settings` at the [configuration](https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-configuration.html) docs